eng/common/scripts/Verify-Links.ps1

```diff
@@ -269,23 +284,38 @@ while ($pageUrisToCheck.Count -ne 0)
 
   $linkUris = GetLinks $pageUri
   Write-Host "Found $($linkUris.Count) links on page $pageUri";
-  
+  $badLinksPerPage = @();
   foreach ($linkUri in $linkUris) {
+    Write-Host "before the replace $linkUri"
+    Write-Host "the commit $branchReplacementName"
     $linkUri = ReplaceGithubLink $linkUri
-
+    Write-Host "after the replace $linkUri"
     $isLinkValid = CheckLink $linkUri
-    if (!$isLinkValid) {
-      $script:badLinks += $linkUri
+    if (!$isLinkValid -and !$badLinksPerPage.Contains($linkUri)) {
+      $badLinksPerPage += $linkUri
     }
     if ($recursive -and $isLinkValid) {
       if ($linkUri.ToString().StartsWith($baseUrl) -and !$checkedPages.ContainsKey($linkUri)) {
         $pageUrisToCheck.Enqueue($linkUri);
       }
     }
   }
+  if ($badLinksPerPage.Count -gt 0) {
+    $badLinks[$pageUri] = $badLinksPerPage
+  }
 }
 
-Write-Host "Found $($checkedLinks.Count) links with $($badLinks.Count) broken"
```

Comment:
We should probably still output some sort of a header so we know what this output is.
```
if ($badLinks.Count -gt 0) {
  Write-Host "Summary of broken links:"
}
```

Reply:
This looks to be missing still.

Reply:
Yes. Just added.